MAUI/TreeView/selection.md

```diff
@@ -0,0 +1,167 @@
+---
+layout: post
+title: Selection in .NET MAUI TreeView control | Syncfusion
+description: Learn here all about Selection support in Syncfusion .NET MAUI TreeView (SfTreeView) control and more.
+platform: .NET MAUI
+control: SfTreeView
+documentation: ug
+---
+
+# Selection in .NET MAUI TreeView (SfTreeView)
+
+This section explains how to perform selection and its related operations in the TreeView.
+
+## UI Selection
+The TreeView allows selecting the items either programmatically or touch interactions by setting the `SelectionMode` property value to other than `None`. The control has different selection modes to perform selection operations as listed as follows.
+
+* `None`: Allows disabling the selection.
+* `Single`: Allows selecting the single item only. When clicking on the selected item, selection not will not be cleared. This is the default value for `SelectionMode`.

```

Comment:
* `Single`: Allows selecting the single item only. When clicking on the selected item, the selection will not be cleared. This is the default value for `SelectionMode`.